adds new sample for displaying cards

LuiseFreese opened the pull request:
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New sample?      | yes
| Related issues?  | n/a

#### What's in this Pull Request?

adds a new sample, hope this is useful. I also tried my best to create the sample.json file so that this sample can get surfaced in the sample solution gallery. 
Let me know if this is ok or if any changes are needed.

LuiseFreese commented:
Thanks for making the adjustments!!

tecchan1107 commented:
Thank you again for the so beautiful sample @LuiseFreese ! I have merged them!
https://github.com/pnp/List-Formatting/tree/master/column-samples/generic-card-with-buttons

Also, when merging, I made the following corrections:

- For user pictures, there is a dedicated operator (getUserImage), so I changed to use that one. (Reference: [Formatting syntax reference - Operators](https://learn.microsoft.com/sharepoint/dev/declarative-customization/formatting-syntax-reference#operators))  
    ![image](https://github.com/user-attachments/assets/c18d9fc9-69f5-4e4e-b6b9-052adaf14b33)

- Changed `Assigned to` in README.md to `Assignedto` to match JSON

Thank you also for creating sample.json! It was very helpful!

LuiseFreese commented:
[heart]         Luise Freese reacted to your message:
________________________________
From: Tetsuya Kawahara ***@***.***>
Sent: Wednesday, October 16, 2024 5:33:32 AM
To: pnp/List-Formatting ***@***.***>
Cc: Luise Freese ***@***.***>; Mention ***@***.***>
Subject: Re: [pnp/List-Formatting] adds new sample for displaying cards (PR #860)


Thank you again for the so beautiful sample @LuiseFreese<https://github.com/LuiseFreese> ! I have merged them!
https://github.com/pnp/List-Formatting/tree/master/column-samples/generic-card-with-buttons

Also, when merging, I made the following corrections:

  *   For user pictures, there is a dedicated operator (getUserImage), so I changed to use that one. (Reference: Formatting syntax reference - Operators<https://learn.microsoft.com/sharepoint/dev/declarative-customization/formatting-syntax-reference#operators>)
image.png (view on web)<https://github.com/user-attachments/assets/c18d9fc9-69f5-4e4e-b6b9-052adaf14b33>

  *   Changed Assigned to in README.md to Assignedto to match JSON

Thank you also for creating sample.json! It was very helpful!

—
Reply to this email directly, view it on GitHub<https://github.com/pnp/List-Formatting/pull/860#issuecomment-2415771360>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AL5FMIT7HCIBIAOQOYIDPG3Z3X3CZAVCNFSM6AAAAABP5GEZZSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMJVG43TCMZWGA>.
You are receiving this because you were mentioned.Message ID: ***@***.***>
